src/HIE/Bios/Environment.hs

```diff
@@ -167,6 +167,17 @@ addCmdOpts cmdOpts df1 = do
        liftIO $ hPutStrLn stderr "cannot set package flags with :seti; use :set"
     -}
 
+makePathsAbsolute :: FilePath -> DynFlags -> DynFlags
```

Comment:
Should this be part of the public APIs? Probably, but then I think a better name could be helpful?